bin/inx-chronicle/src/stardust_inx/error.rs

```diff
@@ -3,7 +3,7 @@
 
 use thiserror::Error;
 
-use super::listener::InxListenerError;
+use super::listener::StardustInxListenerError;
```

Comment:
I wonder if that should just be `stardust::InxListenerError`?

Reply:
I renamed them so we wouldn't have potential naming conflicts later in the event that we do add more protocols that use INX.